registry/securing-exposing-registry.adoc

```diff
@@ -7,11 +7,8 @@ toc::[]
 ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
 
 By default, the {product-title} registry is secured during cluster
-installation so that it serves traffic via TLS. A passthrough route is also
-created by default to expose the service externally.
+installation so that it serves traffic via TLS. 
```

Comment:
```suggestion
installation so that it serves traffic via TLS. 
Unlike previous versions of OpenShift, the registry is not exposed outside of the cluster at installation time.
```